src/Kernel/Services/OrderCreationService.php

```diff
@@ -0,0 +1,107 @@
+<?php
```

Comment:
"Resilience" não faz muito sentido no contexto do código. Trocar tudo que for **resilience** por **retry**